news/9103.feature.rst

```diff
@@ -0,0 +1 @@
+Add CI configuration for `Opendev <https://opendev.org>`__ `Zuul <https://zuul-ci.org/>`__ testing
```

Comment:
Technically this is not a feature in pip but "process" or "trivial". @pradyunsg WDYT?

Reply:
Trivial -- this isn't a user facing change.